build-tools/automation/azure-pipelines.yaml

```diff
@@ -383,7 +383,6 @@ stages:
         testName: Mono.Android_TestsAppBundle
         project: tests/Mono.Android-Tests/Runtime-AppBundle/Mono.Android-TestsAppBundle.csproj
         testResultsFiles: TestResult-Mono.Android_TestsAppBundle-$(ApkTestConfiguration).xml
-        packageType: Aab
```

Comment:
I don't see a mention of this change in the PR notes.

Reply:
The new `RunTestApp` target enables this change. Previously we would call either `DeployTestApks` or `DeployTestAabs` depending on which package type was being used by the test. That was unnecessary as these targets will only run if a `@(TestApk)` or `@(TestAab)` item is provided, respectively. Both targets can be called sequentially and only one will run based on what is set in the test project.